site/includes/footerresources.hbs

```diff
@@ -1,5 +1,6 @@
 <!--[if gte IE 9 | !IE ]><!-->
 <script src="{{#if offline}}{{assets}}/../wet-boew/js{{else}}https://ajax.googleapis.com/ajax/libs{{/if}}/jquery/{{environment.jqueryVersion}}/jquery.js"></script>
+<script>window.jQuery || document.write('<script src="{{assets}}/../wet-boew/js/jquery/{{environment.jqueryVersion}}/jquery.js"><\/script>');</script>
```

Comment:
Without the line I wasn't able to get Jquery to load when blocking Google's Jquery url using Chrome.